src/validateEmail.test.js

```diff
@@ -17,4 +17,38 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   // write more tests here
+  it(`should return 'false' for the email with the characters
+    not from ASCII`, () => {
+    expect(validateEmail('Ыpersonal_info@gmail.com')).toBe(false);
+  });
+
+  it(`should return 'true' for the email with the digits`, () => {
+    expect(validateEmail('personal_info123@gmail.com')).toBe(true);
+  });
+
+  it(`should return 'true' for the email with "_"`, () => {
+    expect(validateEmail('personal_info@gmail.com')).toBe(true);
+  });
+
+  it(`should return 'false' for the email starts with "." `, () => {
+    expect(validateEmail('.personal_info@gmail.com')).toBe(false);
+  });
+
+  it(`should return 'false' for the email without @`, () => {
+    expect(validateEmail('personal_infogmail.com')).toBe(false);
+  });
+
+  it(`should return 'false' for the email where domain starts with "."`, () => {
+    expect(validateEmail('personal_infog@.mail.com')).toBe(false);
+  });
+
+  it(`should return 'false' for the email where
+    domain don't includes "."`, () => {
+    expect(validateEmail('personal_info@gmailcom')).toBe(false);
+  });
+
+  it(`should return 'true' for the email where
+    domain don't includes digits`, () => {
```

Comment:
The description of the test case does not match the expected result. It says that the test should return 'true' for an email where the domain doesn't include digits, but the tested email ('personal_info@gmail1.com') does include a digit in the domain. Please clarify the requirements and adjust the test case accordingly.